src/metabase/cmd/env_var_dox.clj

```diff
@@ -0,0 +1,110 @@
+(ns metabase.cmd.env-var-dox
+  "Code to generate docs for environment variables. You can generate
+  docs by running: `clojure -M:ee:run environment-variables-documentation`"
+  (:require [clojure.java.classpath :as classpath]
+            [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.tools.namespace.find :as ns.find]
+            [clojure.tools.reader.edn :as edn]
+            [metabase.models.setting :as setting]
+            [metabase.util :as u]))
+
+(def env-vars-not-to-mess-with
+  "Flamber advises that people avoid touching these environment variables."
+  (set (edn/read-string (slurp (io/file "src/metabase/cmd/resources/env-vars-to-avoid.edn")))))
+
+(defn- get-settings
+  "Loads all of the metabase namespaces, which loads all of the defsettings,
+  which are registered in an atom in the settings namespace. Once settings are registered,
+  This function derefs that atom and puts the settings into a sorted map for processing."
+  []
+  (for [ns-symb (ns.find/find-namespaces (classpath/system-classpath))
```

Comment:
```suggestion
  (doseq [ns-symb (ns.find/find-namespaces (classpath/system-classpath))
```

You need to use `doseq` here. `for` returns a lazy sequence and doesn't actually do anything until you "consume" that lazy sequence, which you're not doing. So this actually does not do anything. `doseq` on the other hand eagerly goes thru the whole loop, for side effects, which is exactly what you want here